src/preact-render-spy.js

```diff
@@ -164,6 +164,12 @@ const verifyFoundNodes = wrapper => {
   }
 };
 
+const verifyOnlySingleNode = wrapper => {
```

Comment:
:1st_place_medal: thanks for this, we needed it

Reply:
Is it weird if I ask you open a PR with this method and using it in existing methods? That'd be easy to merge in.

Reply:
Sure thing :)

Reply:
Take a look at #69. When we merge that PR I will merge master to this branch :)

Reply:
Done :)